reputation/grpc/service.pb.go

```diff
@@ -7,11 +7,12 @@
 package reputation
 
 import (
+	reflect "reflect"
+	sync "sync"
+
```

Comment:
Not sure this should have been changed :thinking:

Reply:
Fixed.